resources/ark/README.md

```diff
@@ -24,12 +24,14 @@ To add components to the backup pipeline, implement the following go interface:
 type BackupTest interface {
     CreateResources(namespace string)
     TestResources(namespace string)
+    DeleteResources(namespace string) error
```

Comment:
Please unify the approach. If this method returns an error, then `CreateResource` and `TestResources` should do this as well.